src/main/resources/testchipip/csrc/SimDromajoCosim.cc

```diff
@@ -0,0 +1,114 @@
+#include <vpi_user.h>
+#include <svdpi.h>
+
+#include <stdio.h>
+#include <unistd.h>
+
+#include "dromajo_wrapper.h"
+
+// gotten from elaboration
```

Comment:
Generated during Chipyard elaboration?

Reply:
Yes. Ill go ahead and update the comment.